spec/plurimath/math/formula_spec.rb

```diff
@@ -174,14 +174,14 @@
     context "contains log formula string with text" do
       let(:exp) { "log(1+2+3+4)^(\"4 terms\")" }
       it 'converts formula back to Asciimath string for log function' do
-        expect(formula).to eql('log_(1+2+3+4)^"4 terms"')
+        expect(formula).to eql('log1+2+3+4^"4 terms"')
```

Comment:
This change doesn't seem correct.

The original:
```ruby
"log(1+2+3+4)^(\"4 terms\")"
```

And the converted back formula:
```ruby
'log1+2+3+4^"4 terms"'
```

Are rather different in element precedence.